config/moonguard.php

```diff
@@ -32,7 +32,7 @@
         ],
     ],
     'routes' => [
-        'prefix' => 'api',
+        'prefix' => 'moonguard/api',
         'middleware' => 'throttle:api',
     ],
```

Comment:
Maybe is time to make some comments inside the current config file. Similar to what we did with the [Larvis configuration file](https://github.com/taecontrol/larvis/blob/1.x/config/larvis.php).

Reply:
Good idea

Reply:
Added comments to the config file